### PR TITLE
add an example

### PR DESCRIPTION
**Pull request by jospoortvliet**
spectacle x12456

### CODE REVIEW

**Comment by MariusBluem** [comment]
For the lazy customers 😜

File: page-enterprise.php [hunk 1]
```diff
@@ -185,7 +185,7 @@
 		<td class="eenterprise text-center"><a href="#contact">Contact us!</a></td>
 	</tr>
 </table>
-<p class="epricevat">Prices are in Euro and net/excl. VAT.</p>
+<p class="epricevat">Prices are in Euro and net/excl. VAT.<br />As a quick pricing example, 150 users on a Pro subscription would cost 100*50 + 50*30 = Eur 6500/year</p>
```
Comment (MariusBluem) [inline page-enterprise.php]: `Eur` should be `€` or `EUR` 😉

**Comment by jospoortvliet** [comment]
Fixed. I created a table to make it a bit more clear:
spectacle j30859

**Comment by MariusBluem** [comment]
👍

**Comment by jospoortvliet** [comment]
Feedback from Andreas:

* Formatting: i would put the calculation with a gap to the lifecycle support
* less tabs between users and the sums
* At the beginning of the sum rows a would put a + on every row

Going to do that now ;-)

**Comment by jospoortvliet** [comment]
spectacle f29813

**Comment by LukasReschke** [comment]
Maybe add a JS price calculator in the redesign? - Just a thought :)

**Pull request merged**